includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -73,6 +71,9 @@ public function sanitize() {
 		}
 	}
 
+	/**
```

Comment:
Missing function description, param description. We've also been adding `@since` tags.

Reply:
Documented.